build.rs

```diff
@@ -20,7 +20,6 @@ fn main() {
 
     let dst = cmake::Config::new("acquire-common")
         .profile("RelWithDebInfo")
-        .static_crt(true)
```

Comment:
Why do we no longer link against the static version of the C runtime? How does it relate to the changes in this PR?

Reply:
This looks like an artifact from some older experimenting with the MSVC runtime. This change shouldn't be there.